tests/app_test.py

```diff
@@ -0,0 +1,194 @@
+from flask.wrappers import Response
+from controllers import app
+from controllers import db
+from flask_sqlalchemy import SQLAlchemy
+from services.data_service import DataService
+from datetime import datetime
+from models import *
+import json
+import os
+
+tester = app.app.test_client()
+
+QA_MODEL_CONTROLLER_VERSION = 'v1'
+
+def test_root():
+    response = tester.get("/", content_type="html/text")
+
+    assert response.status_code == 200
+    assert response.data == b'"The TweetQA Api is running!"\n'
+
+def mock_db_setup():
+    app.app.config['TESTING'] = True
+    app.app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///test.db'
+    db.drop_all()
+    db.create_all()
+    db.session.commit()
+
+def mock_db_teardown():
+    db.session.close()
+    os.remove('controllers/test.db')
+
+def test_data_service_creates_data():
+    '''TC-001: The DataService object can create a datum entity in the database'''
+    mock_db_setup()
```

Comment:
Would be worth looking into pytest configuring, https://docs.pytest.org/en/latest/reference/reference.html#_pytest.hookspec.pytest_configure

You can setup and teardown the mock database just once at the beginning of all tests and then again at the end of all tests using

pytest_sessionstart(session)[source]
Called after the Session object has been created and before performing collection and entering the run test loop.

and

pytest_sessionfinish(session, exitstatus)[source]
Called after whole test run finished, right before returning the exit status to the system.